benchmarks/cugraph/standalone/cugraph_bulk_sampling.py

```diff
@@ -0,0 +1,736 @@
+                                    num_labels=256,
+                                    labeled_percentage=0.001,
+                                    persist=False,
+                                    add_edge_types=False):
+    """
+    Entry point for the benchmark.
+
+    Parameters
+    ----------
+    dataset: str
+        The dataset to sample.  Can be rmat_{scale}_{edgefactor}, or the name of an ogb dataset.
+    output_path: str
+        The output path, where samples and metadata will be stored.
+    seed: int
+        The random seed.
+    batch_size: int
+        The batch size (number of input seeds in a single sampling batch).
+    seeds_per_call: int
+        The number of input seeds in a single sampling call.
+    fanout: list[int]
+        The fanout.
+    reverse_edges: bool
+        Whether to reverse edges when constructing the graph.
+    dataset_dir: str
+        The directory where datasets are stored (only for ogb datasets)
+    replication_factor: int
+        The number of times to replicate the dataset.
+    num_labels: int
+        The number of random labels to generate (only for rmat datasets)
+    labeled_percentage: float
+        The percentage of the data that is labeled (only for rmat datasets)
+        Defaults to 0.001 to match papers100M
+    persist: bool
+        Whether to aggressively persist data in dask in attempt to speed up ETL.
+        Defaults to False.
+    add_edge_types: bool
+        Whether to add edge types to the edgelist.
+        Defaults to False.
+    """
+    print(dataset)
+    if dataset[0:4] == 'rmat':
+        dask_edgelist_df, dask_label_df, node_offsets, edge_offsets, total_num_nodes = \
+            generate_rmat_dataset(
+                dataset,
+                reverse_edges=reverse_edges,
+                seed=seed,
+                labeled_percentage=labeled_percentage,
+                num_labels=num_labels,
+                persist=persist,
+                add_edge_types=add_edge_types
+            )
+
+    else:
+        dask_edgelist_df, dask_label_df, node_offsets, edge_offsets, total_num_nodes = \
+            load_disk_dataset(
+                dataset,
+                dataset_dir=dataset_dir,
+                reverse_edges=reverse_edges,
+                replication_factor=replication_factor,
+                persist=persist,
+                add_edge_types=add_edge_types
+            )
+
+    num_input_edges = len(dask_edgelist_df)
+    print(
+        f"Number of input edges = {num_input_edges:,}"
+    )
+
+    G = construct_graph(
+        dask_edgelist_df
+    )
+    print('constructed graph')
+
+    input_memory = G.edgelist.edgelist_df.memory_usage().sum().compute()
+    print(f'input memory: {input_memory}')
+
+    now = datetime.now()
+    timestring = datetime.strftime(now, '%Y-%m-%d-%H:%M:%S')
+    output_subdir = os.path.join(output_path, f'{dataset}[{replication_factor}]_b{batch_size}_f{fanout}_{timestring}')
```

Comment:
Having a timing string attached here is making scripting non straightforward for downstream consumption of this in bulk sampling script .  Would it be possible for us to remove this ? 

```suggestion
    output_subdir = os.path.join(output_path, f'{dataset}[{replication_factor}]_b{batch_size}_f{fanout})
```